### ISSUE TITLE
Predictions not matching by XGBoost and PMML file

### ISSUE THREAD

**Issue by Shashwat3011** [iamDecode/sklearn-pmml-model#58]
I am creating an XGB Classifier object using this snippet

xgb = XGBClassifier(objective = 'binary:logistic',
                    booster = 'gbtree',
                    learning_rate = 0.1,
                    subsample = 0.8,
                    random_state = 42,
                    colsample_bytree = 0.8,
                    gamma = 0.5,
                    max_depth = 6,
                    min_child_weight = 4,
                    n_estimators = 130,
                    missing = -9999.01)

xgb.fit(x_train.copy(), y_train, sample_weight = sample_weights)

Then Calling a PMML Pipeline by 

pipeline = PMMLPipeline([
  ("classifier", XGBClassifier( objective = 'binary:logistic',
                                booster = 'gbtree',
                                learning_rate = 0.1,
                                subsample = 0.8,
                                random_state = 42,
                                colsample_bytree = 0.8,
                                gamma = 0.5,
                                max_depth = 6,
                                min_child_weight = 4,
                                n_estimators = 130,
                                missing = -9999.01))
])

pipeline.fit(x_train.copy(), y_train, classifier__sample_weight = sample_weights)
pipeline.configure(compact = False)

And I am trying to export my model to pipeline using this snippet:

sklearn2pmml(pipeline, "XGBClassifierdefault.pmml")

But after exporting when I am trying to load this exported PMML file, predictions are not matching in XGB Vs PMML file generated

from sklearn_pmml_model.ensemble import PMMLGradientBoostingClassifier
xgb_PMML = PMMLGradientBoostingClassifier(pmml = 'XGBClassifier-default.pmml')

Can you pelase help

**Comment by iamDecode** [comment]
> TLDR: The source of misprediction resides inside the [Sklearn-PMML-Model](https://github.com/iamDecode/sklearn-pmml-model) package.

It was concluded that the mismatch was not caused by this library, but a downstream dependency instead. Closing this issue.
